### PR TITLE
Remove dependency to safer-buffer since we have dropped support for old node

### PR DESCRIPTION
**Pull request by TheThing**
Also fix husky in some windows environment

### CODE REVIEW

**Comment by ashtuchkin** [comment]
Sorry for my ignorance, but who exactly has dropped support for old node?

**Comment by TheThing** [comment]
Uhh, next version has. It's literally in the package.json: https://github.com/ashtuchkin/iconv-lite/blob/next/package.json#L22
And according to safer-buffer: https://github.com/ChALkeR/safer-buffer/blob/master/Porting-Buffer.md#variant-1 we don't need to use it anymore.

**Comment by ashtuchkin** [comment]
ahh I see, I didn't realize you're using 'next' branch.

**Comment by ashtuchkin** [comment]
ok merged, thank you!

**Comment by TheThing** [comment]
Any chance we could get a pre-release of next or maybe official next release?
Would be nice to start using this in production environment. Were there any breaking changes with the introductions of backend?
I wouldn't mind helping with whatever is needed to get `next` into release :)

File: package.json [hunk 1]
```diff
@@ -31,7 +31,7 @@
     },
     "husky": {
         "hooks": {
-            "pre-commit": "lint-staged --concurrent=false"
+            "pre-commit": "npx lint-staged --concurrent=false"
```
Comment (TheThing) [inline package.json:34]: Fixed some problems when running, at least, in windows environment.

**Pull request merged**